app/controllers/api/v1/applications_controller.rb

```diff
@@ -1,14 +1,40 @@
-module Api::V1
-  # Applications controller for access via the API
-  class ApplicationsController < ApiController
-    def index
-      attributes = params.slice(:page, :application_number, :listing, :first_name, :last_name, :submission_type)
-      applications = application_service.applications(attributes)
-      render json: applications
-    end
+module Api
+  module V1
+    # RESTful JSON API for application actions
+    class ApplicationsController < ApiController
+      before_action :authenticate_user!
+
+      def index
+        attributes = params.slice(:page, :application_number, :listing, :first_name, :last_name, :submission_type)
+        applications = application_service.applications(attributes)
+        render json: applications
+      end
+
+      def update
+        response = rest_application_service.update(application_params.merge(id: params[:id]))
+        if response
+          render json: true
+        else
+          render status: 422, json: false
+        end
+      end
+
+      private
+
+      def application_params
+        params.require(:application).permit(
+          :id,
+          :total_monthly_rent,
+        )
+      end
```

Comment:
Here we only allow the set of params that we actually need in the application REST API update requests we have so far. So far, we only make one kind of application REST API update, and that's to update just the total monthly rent field.